.github/workflows/itest-deploy.yml

```diff
@@ -28,14 +28,14 @@ jobs:
             ALLOWED_USERS=$(cat allowed_users.txt)
             COMMENT_BODY="${{ github.event.comment.body }}"
             COMMENT_USER="${{ github.event.comment.user.login }}"
-            ITEST_DEPLOY_KEY="${{ secrets.ITEST_DEPLOY_KEY }}"
+            ITEST_DEPLOY_TRIGGER="${{ vars.ITEST_DEPLOY_TRIGGER }}"
 
-            if [[ "$COMMENT_BODY" == "$ITEST_DEPLOY_KEY" && "$ALLOWED_USERS" == *"$COMMENT_USER"* ]]; then
+            if [[ "$COMMENT_BODY" == "$ITEST_DEPLOY_TRIGGER" && "$ALLOWED_USERS" == *"$COMMENT_USER"* ]]; then
```

Comment:
We should check that `$COMMENT_BODY` has `$ITEST_DEPLOY_TRIGGER` in it, rather than requiring an exact match. So that a comment like this would work:
```
Dear robots, please test this.
```

You can use:
```
if [[ contains($COMMENT_BODY, $ITEST_DEPLOY_TRIGGER)
```

See: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#contains